frontend/src/model/beta/Releases.tsx

```diff
@@ -1,18 +1,24 @@
-import { Box, Button, Stack } from '@mui/material'
+import { Box, Button, Stack, Typography } from '@mui/material'
 import { useEffect, useState } from 'react'
+import EmptyBlob from 'src/common/EmptyBlob'
 
+import { useGetReleasesForModelId } from '../../../actions/release'
+import { ModelInterface } from '../../../types/types'
 import DraftNewReleaseDialog from '../DraftNewReleaseDialog'
 import ModelReleaseDisplay from '../ModelReleaseDisplay'
 
-export default function Releases({ model }: { model: any }) {
+export default function Releases({ model }: { model: ModelInterface }) {
   const [latestRelease, setLatestRelease] = useState<string>('')
   const [openDraftNewRelease, setOpenDraftNewRelease] = useState(false)
 
+  const { releases, isReleasesLoading } = useGetReleasesForModelId(model.id)
+
   useEffect(() => {
-    if (model && model.releases) {
-      setLatestRelease(model.releases.sort((a, b) => a.semver < b.semver)[0].semver)
+    if (model && releases) {
+      const sortedReleases = releases.sort((a, b) => (a.semver < b.semver ? 1 : b.semver > a.semver ? -1 : 0))
```

Comment:
With complex sorting logic like this I find it's usually pretty unreadable. Therefore I prefer to pull the logic out into a function and give it a descriptive name so that it's easy to read at a glance. I believe this is sorting by version descending, so have assumed that for the following example, but please change that if I'm incorrect.

Also, nested ternaries are always worth avoiding again for readability.
```
// Probably worth pulling this out into a utils file
const sortByReleaseVersionDescending = (a: ReleaseInterface, b: ReleaseInterface) => {
  if (a.semver < b.semver) return 1
  if (b.semver > a.semver) return -1
  return 0
}

const sortedReleases = releases.sort(sortByReleaseVersionDescending)
```